app/serializers/event_type_serializer.rb

```diff
@@ -4,5 +4,5 @@ class EventTypeSerializer
   include FastJsonapi::ObjectSerializer
 
   set_type :event_type
-  attributes :name
+  attribute :name, &:external_id
```

Comment:
This should be `:title` instead of `:name` now, or?

Reply:
It was missing title. Not sure how to handle the name/external id thing properly. When an app registers an event type, it has to provide an `id` and we store that as `external_id`. I wonder if we should show that on `#show`

Reply:
Ah, so the `external_id` shown as `name`. Was confused, since the migrations drop the name.
Atm. it is not need in the UI, the name and id are enough for it to show it, but in cases of adding and updating apps and their data it would probably make sense to have it shown.